src/Morphic-Base/WindowManager.class.st

```diff
@@ -0,0 +1,202 @@
+"
+I manage the native window, when i was create i open a window with a taskbar, if you create a window, the window is add at the taskbar.
+
+
+    Instance Variables
+	allWindow:		<Object>
+	allWindowState:		<Object>
+	allWorld:		<Object>
+	taskBar:		<Object>
+	taskBarWindow:		<Object>
+
+
+    Implementation Points
+"
+Class {
+	#name : #WindowManager,
+	#superclass : #Morph,
+	#instVars : [
+		'taskBar',
+		'allWindow',
+		'taskBarWindow',
+		'rootSave',
+		'allAttributes'
+	],
+	#category : #'Morphic-Base'
+}
+
+{ #category : #accessing }
+WindowManager >> allAttributes [
+	^ allAttributes
+]
+
+{ #category : #accessing }
+WindowManager >> allSaveRoot [
+	
+	^ rootSave 
+]
+
+{ #category : #accessing }
+WindowManager >> allWindow [ 
+
+	^ allWindow 
+]
+
+{ #category : #accessing }
+WindowManager >> closeAllWindow [
+
+	
+	(allWindow size) to: (1) by: (-1) do: [ :m | (allWindow at: m) delete  ].
+	taskBar updateTasks .
+]
+
+{ #category : #accessing }
+WindowManager >> closeAllWindowToLeft: aWindow [
+
+	| index |
+	index := allWindow indexOf: aWindow.
+	(index - 1 )  to: 1 by: (-1) do: [ :m | (allWindow at: m) delete  ].
+	taskBar updateTasks .
+]
+
+{ #category : #accessing }
+WindowManager >> closeAllWindowToRight: aWindow [
+
+	| index |
+	index := allWindow indexOf: aWindow.
```

Comment:
pass a formatting

allWindow => allWindows => windows

put a comment in this method